README.md

```diff
@@ -402,6 +402,21 @@ It'll generate a markdown file if everything went fine. That's where all this cr
 
 And now your crate should be completely documented as expected!
 
+If you defining traits manually you can add them to "Implements" section for classes and interfaces:
+```toml
+[[object]]
+name = "Gtk.Assistant"
+status = "generate"
+#add link to trait from current crate
+manual_traits = ["AssistantExtManual"]
+
+[[object]]
+name = "Gtk.Application"
+status = "generate"
+#add link to trait from other crate
+manual_traits = ["gio::ApplicationExtManual"]
```

Comment:
I don't like this very much but don't have a better idea. Ideally we should be able to re-use the information from the `Gir.toml` of `gio` here but we don't have that...